README.md

```diff
@@ -18,6 +18,8 @@ This is a community project so feel free to contribute ideas, submit bugs, fix b
 
 To use the GCM, you can download the [latest installer](<https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases/latest>). To install, double-click Setup.exe and follow the instructions presented.
 
+When prompted to select your terminal emulator for Git Bash you should choose the Windows' default console window. GCM will not prompt you for credentials in a MinTTY setup.
```

Comment:
GCM cannot prompt you for credentials at the command line in a MinTTY setup. If you prefer to use MinTTY with the GCM, you will need to set `git config credential.modalprompt true` before using the GCM.

Please see #111 for details.